### PR TITLE
Tbq 5 add logback

### PR DESCRIPTION
**Pull request by 1ulans1**

### CODE REVIEW

**Review by Tsurkan — APPROVED** [review]

**Review by IvanKovalov — APPROVED** [review]

File: src/main/resources/application.properties.example [hunk 1]
```diff
@@ -30,3 +30,18 @@ spring.h2.console.settings.trace=false
...
+
+# logback properties
+##################################################################
+logging.file.name=logs/spring-boot-logging.log
+
+#The maximum number of archive log files to keep
+logging.logback.rollingpolicy.max-history=7
+
+#The maximum size of log file before it is archived.
+logging.logback.rollingpolicy.max-file-size=10MB
```
Comment (Sekator778) [inline src/main/resources/application.properties.example:42]: 1MB

Comment (Sekator778) [reply src/main/resources/application.properties.example:42]: max size

File: src/main/resources/application.properties.example [hunk 2]
```diff
@@ -30,3 +30,18 @@ spring.h2.console.settings.trace=false
...
+
+#The maximum amount of size log archives can take before being deleted.
+logging.logback.rollingpolicy.total-size-cap=100MB
```
Comment (Sekator778) [inline src/main/resources/application.properties.example:45]: also need reduce to 10MB

Comment (Sekator778) [reply src/main/resources/application.properties.example:45]: 1

**Review by Sekator778 — COMMENTED** [review]
reduce size files

**Comment by Sekator778** [comment]
why do we log in a file of this size ?

**Comment by 1ulans1** [comment]
because size was not specified in the assignment, and I chose on my own

File: src/main/resources/application.properties.example (hunk 1 above)
Comment (1ulans1) [reply src/main/resources/application.properties.example:42]: why this size?

File: src/main/resources/application.properties.example (hunk 2 above)
Comment (1ulans1) [reply src/main/resources/application.properties.example:45]: why this size?

**Pull request merged**